# Representation of a pedestrian state in the SML World

#string class_name
int32 pedestrian_id
float64 x
float64 y
float64 yaw
